libs/designer/src/lib/ui/panel/templatePanel/quickViewPanel/quickViewPanel.tsx

```diff
@@ -8,14 +8,23 @@ import { TemplatesPanelContent } from '@microsoft/designer-ui';
 import { getQuickViewTabs } from '../../../../core/templates/utils/helper';
 import Markdown from 'react-markdown';
 
+export interface QuickViewTabProps {
```

Comment:
This is not tabprops, maybe you can name it as TemplateContext because you are using this for defining the context in the logging statements and we can add more details in this later on if needed